src/site/stages/build/drupal/graphql-tome-differences/block_content.md

```diff
@@ -0,0 +1,118 @@
+# block_content
+
+Tome returned **28** records.
+
+
+## Missing properties in Tome:
+
+1. `"fieldLinkSummary": "Download letters like your eligibility or award letter for certain benefits."` (This is found from the reference `field_promo_link`, which means it's not missing! Yay! 😎)
+1. `"url": { "path": "/records/download-va-letters" },`
+
+## Example GraphQL response
+
+```json
+{
+  "entity": {
+    "fieldLink": {
+      "url": {
+        "path": "/records/download-va-letters"
+      },
+      "title": "Confirm your VA benefit status",
+      "options": []
```

Comment:
I've found that `options` is never _not_ an empty array in `pages.json` using this:

```
rg "\"options\": \[\]" pages.json
```

Additionally, we only ask for `options` 4 times in the graphql:

```
graphql [filter-cms-export●●] % rg "options"
homePage.graphql.js
71:                      options

navigation-fragments/header.nav.graphql.js
47:                          options

paragraph-fragments/linkTeaser.paragraph.graphql.js
11:      options

landingPage.graphql.js
43:              options
```

- [homePage](https://github.com/department-of-veterans-affairs/vets-website/blob/ea560e7851de773646f5d147492a1bbe1bee513d/src/site/stages/build/drupal/graphql/homePage.graphql.js#L71)
- [navigation-fragments/header.nav](https://github.com/department-of-veterans-affairs/vets-website/blob/ea560e7851de773646f5d147492a1bbe1bee513d/src/site/stages/build/drupal/graphql/navigation-fragments/header.nav.graphql.js#L47)
- [paragraph-fragments/linkTeaser](https://github.com/department-of-veterans-affairs/vets-website/blob/ea560e7851de773646f5d147492a1bbe1bee513d/src/site/stages/build/drupal/graphql/paragraph-fragments/linkTeaser.paragraph.graphql.js#L11)
- [landingPage](https://github.com/department-of-veterans-affairs/vets-website/blob/ea560e7851de773646f5d147492a1bbe1bee513d/src/site/stages/build/drupal/graphql/landingPage.graphql.js#L43)

Lastly, the only template it seems to be used in is `health_care_region_page.drupal.liquid`, but this is just as example data.  I think it is ok that the tome-sync export doesn't provide this.